app.py

```diff
@@ -53,14 +53,21 @@ def upload_image():
         # Add image to document storage
         document_storage.add_image(file_path)
         
+        
+        global SHOULD_GET_DATA 
+        SHOULD_GET_DATA = False
```

Comment:
Unsure why its in so many places.

Reply:
tried to make it the most logical possible, might have over used it